src/components/communities/JoinCommunityPage.tsx

```diff
@@ -114,22 +111,22 @@ export default function JoinCommunityPage({
         width="100%"
       >
         <Box alignItems="center">
-          <ThemedIcon
-            name="cross"
-            type="entypo"
-            size={32}
-            reverse
+          <ThemedIconButton
+            name="x"
+            type="feather"
+            size={64}
             onPress={() => navigation.pop()}

```

Comment:
have we not connected this to the backend yet?

Reply:
This just backs the user to the previous page. The check mark button is the only one that needs functionality

Reply:
I see I thought this the button that needs functionality. mb full diff didnt render